NOTICE

```diff
@@ -0,0 +1,7 @@
+Apache SINGAEasy
+Copyright 2017 The Apache Software Foundation
```

Comment:
Copyright should be NUS DBSystem Group.
SINGAEasy is not an apache project.. 
so it is not donated to any org.

Reply:
?